Automattic/jetpack
# Move videopress options

#### Changes proposed in this Pull Request:

* Migrates VideoPress options to the package

#### Other information:


#### Jetpack product discussion


pe4Cmx-40-p2
#### Does this pull request change what data or activity we track or use?


No

#### Testing instructions:






* Go to Jetpack > Settings
* Search for videopress
* tweak videopress options
* Make sure all works fine (options are persisted)



---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1202561463041776
